pip_compile/scripts/entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
+set -eux
+
+BUILD_PATH="/.pip-requirements/build"
+MOUNT_PATH="/.pip-requirements/mount"
+
+diffcp() {
+  # Copy files only when their contents have changed (prevents Tilt reload)
+  cmp -s "$1" "$2" || cp "$1" "$2" && echo "No changes made to $1."
+}
+
+# Create and enter build directory
+# (reduce pip-compile comments to filenames)
+mkdir -p "$BUILD_PATH"
+cd "$BUILD_PATH"
+
+# Copy source files to build directory
+cp $MOUNT_PATH/*.in ./
+cp $MOUNT_PATH/*.txt ./
+
+# Compile requirements
+for filepath in "$BUILD_PATH"/*.in; do
+  requirements=$(basename "$filepath")
+  pip-compile --generate-hashes --allow-unsafe "$requirements"
```

Comment:
A use-case that the current pip_compile_button extension supports is adding additional arguments to pip-compile. For example, if I want to update a sub-dependency, I can edit my Tiltfile to pass the [`--upgrade-package` argument](https://github.com/jazzband/pip-tools#updating-requirements) temporarily, then remove it when it's done.